internal/service/streaminstance/model_stream_instance.go

```diff
@@ -49,6 +59,15 @@ func NewTFStreamInstance(ctx context.Context, apiResp *admin.StreamsTenant) (*TF
 		dataProcessRegion = returnedProcessRegion
 		diags.Append(diagsProcessRegion...)
 	}
+	var streamConfig = types.ObjectNull(StreamConfigObjectType.AttrTypes)
+	apiStreamConfig := apiResp.StreamConfig
+	if apiStreamConfig != nil && apiStreamConfig.Tier != nil {
```

Comment:
you may want to use GetXXX methods in admin structs like: if apiStreamConfig.GetTier() != nil

Reply:
In this case I am checking if the attribute is set and since it's a pointer I can directly do it as it is now